tests/integration/test_leases.py

```diff
@@ -251,14 +251,16 @@ def test_authorized_create_request(self, auth_headers):
             }
         for _, role in auth_headers.items():
             response = self.client.post('/api/lease', json=payload, headers=role)
-            assert is_valid(response, 200)
+            assert is_valid(response, 201)
 
     def test_authorized_delete_request(self, auth_headers):
+        id = 1
         for _, role in auth_headers.items():
-            response = self.client.delete('/api/lease/1', headers=role)
+            response = self.client.delete('/api/lease/{}'.format(id), headers=role)
             assert is_valid(response, 200)
+            id += 1
```

Comment:
Thanks for catching. I suppose this wont be an issue as long as we have more seeds then roles. I winced when I wrote these for loops here.

Reply:
FYI: I often see a 204 returned on a delete. However, we are returning a json message so this does seem more appropriate. Thanks!